src/pip/_internal/locations/_distutils.py

```diff
@@ -3,6 +3,14 @@
 # The following comment should be removed at some point in the future.
 # mypy: strict-optional=False
 
+# If pip's going to use distutils, it should not be using the copy that setuptools
+# might have injected into the environment. The least-worst option for ensuring this
+# seems to be to try to remove the injected shim, if it's injected.
```

Comment:
```suggestion
# seems to be to try to remove the injected shim, if it's injected. Ref pypa/pip#8761.
```

Reply:
We should also mention this can be removed when the distutils location backend is dropped, after we drop Python 3.9 support.

Reply:
> We should also mention this can be removed when the distutils location backend is dropped, after we drop Python 3.9 support.

That's implied by the fact that this lives within the distutils location backend.

> Ref pypa/pip#8761

Fair enough -- I added this reference, as a full link and rephrased the comment to provide relevant context.